packages/office-ui-fabric-react/src/components/Keytip/Keytip.styles.ts

```diff
@@ -0,0 +1,77 @@
+import { IKeytipStyleProps, IKeytipStyles } from './Keytip.types';
+import { ICalloutContentStyleProps, ICalloutContentStyles } from '../../Callout';
+import { IStyleFunction, IPoint } from '../../Utilities';
+
+export const getStyles = (props: IKeytipStyleProps): IKeytipStyles => {
+  const { theme, disabled, visible } = props;
+  return {
+    container: [
+      {
+        backgroundColor: theme.palette.neutralDark
+      },
+      disabled && {
+        opacity: 0.5,
+      },
+      !visible && {
+        visibility: 'hidden',
+        opacity: 0
+      }
+    ],
+    root: [{
+      textAlign: 'center',
+      paddingLeft: 3,
+      paddingRight: 3,
+      backgroundColor: theme.palette.neutralDark,
+      color: theme.palette.neutralLight,
+      minWidth: 11,
+      lineHeight: 17,
```

Comment:
units please
does out platform default this to pixels? if so then that doesn't match normal CSS which defaults to EMs for line-height @dzearing 
(although i guess padding/margin/etc default to error state instead of defaulting to px)

Reply:
Will do, it should be in pixels